src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -275,6 +277,60 @@ namespace OpenLoco::Ui::PromptBrowse
         *scrollHeight = window->row_height * _numFiles;
     }
 
+    // 0x004464F7
+    static void onScrollMouseDown(window* self, int16_t x, int16_t y, uint8_t scrollIndex)
+    {
+        auto index = y / self->row_height;
+        if (index > _numFiles)
+            return;
+
+        Audio::playSound(Audio::sound_id::click_down, self->x + (self->width / 2));
+
+        file_entry entry = _files[index];
+
+        // Clicking a directory, with left mouse button?
+        if (Input::state() == Input::input_state::scroll_left && entry.is_directory())
+        {
+            appendDirectory(entry.get_name().data());
+            self->invalidate();
+            return;
+        }
+
+        // Clicking a file, with left mouse button?
+        if (Input::state() == Input::input_state::scroll_left)
+        {
+            // Copy the selected filename without extension to text input buffer.
+            strncpy(_text_input_buffer, entry.get_name().data(), std::size(_text_input_buffer));
+
+            // Continue processing for load/save.
+            processFileForLoadSave(self);
+        }
+
+        // Clicking a file, with right mouse button
```

Comment:
```suggestion
        // Clicking a file, with right mouse button
```
@duncanspumpkin Like this?

Reply:
yes